test/unit/parachain/asset-registry.test.ts

```diff
@@ -5,11 +5,17 @@ import { StorageKey, u32 } from "@polkadot/types";
 import { OrmlAssetRegistryAssetMetadata } from "@polkadot/types/lookup";
 import { DefaultAssetRegistryAPI } from "../../../src/";
 import { AssetRegistryMetadataTuple } from "@interlay/interbtc/parachain/asset-registry";
+import * as allThingsEncoding from "../../../src/utils/encoding";
 
 describe("DefaultAssetRegistryAPI", () => {
     let api: ApiPromise;
     let assetRegistryApi: DefaultAssetRegistryAPI;
     let mockMetadata: OrmlAssetRegistryAssetMetadata;
+    let mockStorageKey: StorageKey<[u32]>;
+    // scraped from integration tests
+    const scrapedStorageKey =
+        "0x6e9a9b71050cd23f2d7d1b72e8c1a625b5f3822e35ca2f31ce3526eab1363fd25153cb1f00942ff401000000";
```

Comment:
I think we can construct the expected value instead of having this magic value, there is some info of how to do this in the "Storage Keys" section of this blog: https://www.shawntabrizi.com/substrate/querying-substrate-storage-via-rpc/

Reply:
> I think we can construct the expected value instead of having this magic value, there is some info of how to do this in the "Storage Keys" section of this blog: https://www.shawntabrizi.com/substrate/querying-substrate-storage-via-rpc/

Thanks for that: yes! 
I want to review this at some point anyway. I think with the mocking I have added later, this isn't really needed anymore and can use dummy values as long as they don't break the format.